.github/workflows/ci.yml

```diff
@@ -53,7 +53,15 @@ jobs:
       - run: julia -e 'using InteractiveUtils; versioninfo(verbose=true)'
       - uses: julia-actions/cache@v1
       - uses: julia-actions/julia-buildpkg@v1
-      - uses: julia-actions/julia-runtest@v1
+      #- uses: julia-actions/julia-runtest@v1
+      - name: Run tests without coverage
+        uses: julia-actions/julia-runtest@v1
+        with:
+          coverage: false
+      - name: Run tests with coverage
+        uses: julia-actions/julia-runtest@v1
+        with:
+          coverage: true
       - uses: julia-actions/julia-processcoverage@v1
```

Comment:
IIRC, the CI job file from the `main` branch is used. Otherwise, people could inject arbitrary stuff into CI pipelines in PRs. You could try just pushing the new CI job file to `main` and see what happens after merging `main` into this PR branch.

Reply:
That makes sense. I noticed that there are also some error messages in the CI run related to the cache:
https://github.com/trixi-framework/TrixiShallowWater.jl/actions/runs/7259069286/job/19775508327#step:21:7
This seems to be due to missing permissions. Can you tell if this is a problem or will this go away when the new CI job file is in main?